developers/weaviate/client-libraries/python/v3_v4_migration.md

```diff
@@ -177,19 +198,18 @@ import ManageDataCodeV3 from '!!raw-loader!/_includes/code/howto/manage-data.rea
   </TabItem>
 </Tabs>
 
-### Collections, classes and schemas
+### Nomenclature changes
 
-The focus on [collections](/developers/weaviate/config-refs/schema) in the v4 client is related to some corresponding changes in Weaviate Core:
+Some of the nomenclature within the Weaviate ecosystem is changing, and the client nomenclature has changed accordingly:
 
-- Old-style "class" objects are "collections" now.
-- An old "schema" is a "collection configuration" now.
-- A "property schema" is a sub-set of the collection configuration.
+- A "class", in the sense of a set of objects, is a "collection".
+- The Weaviate "schema" can be accessed through "collection configuration".
 
-There are other Weaviate Core changes as well. The new python client is designed to support these changes. Expect to find differences in the way you interact with schemas, collection configurations, and property configurations.
+Due to the architectural changes as well as evolving nomenclature, expect to find differences in the way you interact with Weaviate.
```

Comment:
Consider:
Due to the architectural changes as well as changing terminology, expect to find differences in the way you interact with Weaviate.

Reply:
Updated